docker-entrypoint.sh

```diff
@@ -32,6 +32,7 @@ while :; do
   if [ ! -z "${NETWORK_BRIDGE_IP}" ]; then
     break
   fi
+  echo "Waiting for ip address on interface ${NETWORK_BRIDGE_IP}"
```

Comment:
Nitpicking, all messages printed by the program are sentences. Please add a full stop at the end of the message.